lib/Conversation/index.js

```diff
@@ -63,7 +65,8 @@ class Conversation extends Component {
     const showComments = this.state.showComments || this.props.showComments;
     const additionalClasses = cx(this.props.className, {
       'is-active': showComments,
-      'is-resolved': this.props.resolved
+      'is-non-interactive':
```

Comment:
The user can still click on it to change state? Does this class name still make sense?